hummingbot/market/celo/celo_cli.py

```diff
@@ -0,0 +1,57 @@
+import subprocess
+from subprocess import CalledProcessError
+from decimal import Decimal
+
+
+symbols_map = {"CGLD": "gold", "CUSD": "usd"}
+
+
+def command(command_text: str):
+    try:
+        output = subprocess.check_output(command_text, stderr=subprocess.STDOUT, shell=True)
+        output = output.decode("utf-8").strip()
+        if output == "":
+            output = None
+        return output
+    except CalledProcessError as e:
+        raise Exception(e.output.decode("utf-8").split("\n")[0])
+
+
+class CeloCli:
```

Comment:
The current convention in our codebase should be `CeloCLI`. Acronyms can stay as capital letters.